laplace/baselaplace.py

```diff
@@ -791,7 +789,7 @@ def _init_H(self):
         self.H = torch.zeros(self.n_params, self.n_params, device=self._device)
 
     def _curv_closure(self, X, y, N):
-        return self.backend.full(X, y, N=N)
+        return self.backend.full(X, y)
```

Comment:
Why does this have to be removed? I think there is reason for having it in case of subset-of-data approximations and it might be already used in Asdl. I would be in favor of not changing it unless necessary.